packages/api-clients/open-api/bffApi.yml

```diff
@@ -11347,7 +11347,7 @@ components:
       type: object
       description: contains the expected payload for purpose creation.
       properties:
-        eserviceId:
+        eServiceId:
```

Comment:
@Carminepo2 does this bring changes to frontend?

Reply:
hum. I probably made this change to align with the purpose open API spec but in reality, we use almost everywhere `eserviceId`. So I can revert this change!

Reply:
Good catch Rob! @MalpenZibo could you revert? 🙏